src/Components/Onboarding/Views/OnboardingThankYou.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-relative-import-paths/no-relative-import-paths */
```

Comment:
☝️

Reply:
Like... I would own it if I did it, but I did not add this line. I'm guessing that someone's VSCode thought this was an ok autofix option?

Reply:
this has happened to me often, where i'll accidentally right click and disable, but beside that just thinking how liveshare works, how complicated it is... who knows?